Running a specific version in HA: link procedure on ssh add-on set up (#40119)
home-assistant/home-assistant.github.io: source/_includes/common-tasks/specific_version.md

{% if page.installation == "os"%}

> To upgrade to a specific version, you can use the CLI (command line). The example below shows how to upgrade to `{{current_version}}`. To learn how to get started with the command line in Home Assistant, refer to the section [](/common-tasks/os/#installing-and-using-the-ssh-add-on).

Author: ```suggestion
To upgrade to a specific version, you can use the CLI (command line). The example below shows how to upgrade to `{{current_version}}`. To learn how to get started with the command line in Home Assistant, refer to the [SSH add-on setup instructions](/common-tasks/os/#installing-and-using-the-ssh-add-on).
```

> To upgrade to a specific version, you can use the CLI (command line). The example below shows how to upgrade to `{{current_version}}`. To learn how to get started with the command line in Home Assistant, refer to the [SSH add-on setup instructions](/common-tasks/os/#installing-and-using-the-ssh-add-on).

Author: ```suggestion
To upgrade to a specific version, you can use the command line (CLI ). The example below shows how to upgrade to `{{current_version}}`. To learn how to get started with the command line in Home Assistant, refer to the [SSH add-on setup instructions](/common-tasks/os/#installing-and-using-the-ssh-add-on).
```

> To upgrade to a specific version, you can use the command line (CLI ). The example below shows how to upgrade to `{{current_version}}`. To learn how to get started with the command line in Home Assistant, refer to the [SSH add-on setup instructions](/common-tasks/os/#installing-and-using-the-ssh-add-on).

Author: ```suggestion
To upgrade to a specific version, you can use the command line (CLI). The example below shows how to upgrade to `{{current_version}}`. To learn how to get started with the command line in Home Assistant, refer to the [SSH add-on setup instructions](/common-tasks/os/#installing-and-using-the-ssh-add-on).
```
